includes/class-cli-commands.php

```diff
@@ -106,8 +106,14 @@ public function find_domains( array $args, array $assoc_args ) {
 	 * default: 302
 	 * ---
 	 *
+	 * [--preserve-parameters=<preserve>]
+	 * : Preserve URL parameters.
+	 * ---
+	 * default: 0
+	 * ---
+	 *
 	 * @subcommand insert-redirect
-	 * @synopsis <from_url_relative> <to_url_absolute> [--code=<code>]
+	 * @synopsis <from_url_relative> <to_url_absolute> [--code=<code>] [--preserve-parameters=<preserve>]
```

Comment:
If the value of this flag is a boolean it doesn't need a value, the presence of `--presereve-parameters` means it's true, while its absence would mean it's false.

WP CLI handles the boolean value for you. It's not super clear from the docs here but it is mentioned: https://make.wordpress.org/cli/handbook/guides/commands-cookbook/#longdesc

```suggestion
	 * @synopsis <from_url_relative> <to_url_absolute> [--code=<code>] [--preserve-parameters]
```